pywinauto/controls/uia_controls.py

```diff
@@ -543,6 +544,16 @@ def is_selected(self):
         """Return True if the ListItem is selected"""
         return self.iface_selection_item.CurrentIsSelected
 
+    #-----------------------------------------------------------
+    def is_checked(self):
+        """Return True if the ListItem is checked"""
+        # Only items supporting Toggle pattern should answer
+        try:
+            res = (self.iface_toggle.ToggleState_On == toggle_state_on)
+        except NoPatternInterfaceError:
+            res = False
```

Comment:
Maybe we shouldn't return `False` silently in this case. I'd suggest to leave this error unhandled or re-raise kind of `RuntimeError`. It should be more clear for users.